elife_api_validator/media_type.py

```diff
@@ -0,0 +1,66 @@
+import re
+
+from elife_api_validator.exceptions import MissingContentType
+
+
+class MediaType(object):
+    params = {}
+    type = ''
+
+    def __init__(self, content_type: str) -> None:
+        self._content_type = content_type
+        if self._content_type:
+            self._parse_content_type(self._content_type)
+        else:
+            raise MissingContentType('Please supply a content type')
+
+    @property
+    def content_type(self) -> str:
+        return self._content_type
```

Comment:
This would be better as `__str__`?

Reply:
Mmm, I would normally use the `@property` convention and unless the end user was going to be using the MediaType as a string a lot (for concatenation or formatting), that said I can't see a problem with changing it here so I have.